example_client/stateful/rest_stateful_client.py

```diff
@@ -77,16 +79,16 @@ def parse_arguments():
         required=False,
         default='rm_lstm4f/test_score_1_10.ark',
         help='Path to reference scores ark file')
-    parser.add_argument(
-        '--rest_address',
-        required=False,
-        default='localhost',
-        help='Specify url to rest service. default:http://localhost')
     parser.add_argument(
         '--rest_port',
         required=False,
         default=5555,
         help='Specify port to rest service. default: 5555')
+    parser.add_argument(
```

Comment:
Why do we need that change? It will be inconsistent with all other examples where we use `address` instead of `url`.

Reply:
For REST we use url in examples.